internal/testutil/acc/atlas.go

```
@@ -27,6 +27,11 @@ func createProject(tb testing.TB, name string) string {
 
 func deleteProject(id string) {
 	_, err := ConnV2().ProjectsApi.DeleteProject(context.Background(), id).Execute()
+	if admin.IsErrorCode(err, "CANNOT_CLOSE_GROUP_ACTIVE_ATLAS_CLUSTERS") {
```

[nit] Not strictly necessary to include, but a low cost of extra 30s wait time instead of pending resources

---

not against, but in which cases will this happen?  in theory all clusters should be deleted before calling this func, except in special circunstances, and we might get the error in that case to investigate what's going on instead of ignoring the error.

(also at the end cleanup job will delete any leftover)

---

> except in special/error circumstances

Exactly! This is when it was useful for me, when adding the new test.

Notice how the error is still logged, it is just an extra attempt

---

I guess the discussion is about if we want the test to fail and see what's going on, or we're ok to let it pass.   I normally prefer to make the test fail so it's obvious that something is going on.  Conversely, if this is something expected from time to time and it's a false positive, ok to retry.

not a blocking comment, feel free to decide in either side

---

I like this idea of failing.
However, I think it is a bit tricky to implement as this is called from the TestMain, let me check

---

Adding to our tech discussion.